test/NLogTarget.Net45.Tests/NLogTargetTests.cs

```diff
@@ -222,7 +222,9 @@ public void TraceHasCustomProperties()
         [TestCategory("NLogTarget")]
         public void GlobalDiagnosticContextPropertiesAreAddedToProperties()
```

Comment:
@snakefoot - consider renaming test: `GlobalDiagnosticContextPropertiesAreAddedToPropertiesIfContextIsSet`.  That's a long method name, but it clarifies the change you made.

This also suggests a test: `GlobalDiagnosticContextPropertiesAreNotAddedToPropertiesIfContextNotSet`. Which is an even longer method name.

Reply:
Think it is a little strange to have unit-test that verifies functionality is not there. Have no opinion about the unit test names. Just happy that they include `GlobalDiagnosticContext` in the name.

Reply:
OK, will leave the decision to @SergeyKanzhelev, it's fine either way.